edge/evtstreams/cpu2evtstreams/README.md

```diff
@@ -42,27 +44,31 @@ hzn exchange node create -n $HZN_EXCHANGE_NODE_AUTH
 hzn exchange node confirm
 ```
 
-6. Deploy (or get access to) an instance of IBM Event Streams that the cpu2evtstreams sample can send its data to. Ensure that the topic `cpu2evtstreams` is created in Event Streams. Using information from the Event Streams UI, `export` these environment variables:
-    - `EVTSTREAMS_API_KEY`
-    - `EVTSTREAMS_BROKER_URL`
-    - `EVTSTREAMS_CERT_ENCODED` **(if using IBM Event Streams in IBM OpenShift Cloud Private)** due to differences in the base64 command set this variable as follows based on the platform you're using:
-        - on **Linux**: `EVTSTREAMS_CERT_ENCODED=“$(cat $EVTSTREAMS_CERT_FILE | base64 -w 0)”`
-        - on **Mac**: `EVTSTREAMS_CERT_ENCODED=“$(cat $EVTSTREAMS_CERT_FILE | base64)”`
-    - `EVTSTREAMS_CERT_FILE` **(if using IBM Event Streams in IBM OpenShift Cloud Private)**
+6. While this service can be used with any kafka based message brokers, if you are using IBM Event Streams and an instance has already been deployed for you, obtain the `event-streams.cfg` file that was created during this process. This file contains all the necessary environment variables for `cpu2evtstreams` to publish data to IBM Event Streams. Set these environment variables in your shell now:
+```bash
+eval export $(cat event-streams.cfg)
+```
+
+7. If you have not done so already, unregister your node before moving on:
+ ```bash
+hzn unregister -f
+```
 
 
 ## <a id=using-cpu2evtstreams-pattern></a> Using the CPU To IBM Event Streams Edge Service with Deployment Pattern
 
-1. Get the user input file for the cpu2evtstreams sample:
+1. Get the user input file for the cpu2evtstreams sample and the policy file for the gps service to run privileged:
 ```bash
-wget https://github.com/open-horizon/examples/raw/master/edge/evtstreams/cpu2evtstreams/horizon/use/userinput.json
+wget https://raw.githubusercontent.com/open-horizon/examples/master/edge/evtstreams/cpu2evtstreams/horizon/use/userinput.json
+wget https://raw.githubusercontent.com/open-horizon/examples/master/edge/services/gps/horizon/node_policy_privileged.json
 ```
 2. Register your edge node with Horizon to use the cpu2evtstreams pattern:
 ```bash
-hzn register -p IBM/pattern-ibm.cpu2evtstreams -f userinput.json
+hzn register -p IBM/pattern-ibm.cpu2evtstreams -f userinput.json -s ibm.cpu2evtstreams --serviceorg IBM -t 120 --policy=node_policy_privileged.json
 ```
+ - **Note**: using the `-s` flag with the `hzn register` command will cause Horizon to wait until agreements are formed and the service is running on your edge node to exit, or alert you of any errors encountered during the registration process. 
 
-3. The edge device will make an agreement with one of the Horizon agreement bots (this typically takes about 15 seconds). Repeatedly query the agreements of this device until the `agreement_finalized_time` and `agreement_execution_start_time` fields are filled in:
+3. View the formed agreement:
 ```bash
 hzn agreement list
 ```
```

Comment:
Since you are using the `-s` flag of hzn register, do they really need to view the agreement?

Reply:
removed